azure-pipelines-frontdoor.yaml

```
@@ -0,0 +1,112 @@
+name: SDS Azure Platform - frontdoor
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+
+parameters:
+  - name: components
+    displayName: Component to Run
+    type: object
+    default:
+      - deployment: 'frontdoor'
+        environment: 'sbox'
+        dependsOn: 'Precheck'
+      - deployment: 'frontdoor'
+        environment: 'dev'
+        dependsOn: 'Precheck'
+      - deployment: 'frontdoor'
+        environment: 'stg'
+        dependsOn: 'Precheck'
+      - deployment: 'frontdoor'
+        environment: 'ithc'
+        dependsOn: 'Precheck'
+      - deployment: 'frontdoor'
+        environment: 'test'
+        dependsOn: 'Precheck'
+      - deployment: 'frontdoor'
+        environment: 'prod'
+        dependsOn: 'Precheck'
+
+  - name: location
+    displayName: Location
+    type: string
+    default: 'UK South'
+    values:
+      - 'UK South'
+      - 'UK West'
+
+variables:
+  - name: tfversion
+    value: 0.14.4
+  - name: project
+    value: application
+  - name: timeoutInMinutes
+    value: '360'
+  - name: agentPool
+    value: 'ubuntu-18.04'
```

don't we want:
```suggestion
    value: 'hmcts-cftptl-agent-pool'
```

https://github.com/hmcts/azure-platform-terraform/blob/master/azure_pipeline-global.yaml#L25